pkgs/development/python-modules/curl-cffi/default.nix

```diff
@@ -81,6 +83,25 @@ buildPythonPackage rec {
   disabledTestPaths = [
     # test accesses network
     "tests/unittest/test_smoke.py::test_async"
+  ]
+  ++ lib.optional stdenv.isDarwin [
+    # "tests/unittest/conftest.py"
+    # "tests/unittest/test_async.py"
+    # "tests/unittest/test_async_session.py"
+    # "tests/unittest/test_cookies.py"
+    # "tests/unittest/test_curl.py"
+    "tests/unittest/test_headers.py"
+    "tests/unittest/test_impersonate.py"
+    "tests/unittest/test_requests.py"
+    "tests/unittest/test_upload.py"
```

Comment:
Also add comments on why these are disabled.

Reply:
Also fixed in the latest push.

Reply:
I don't see comments?

Reply:
```
   disabledTestPaths = [
     # test accesses network
     "tests/unittest/test_smoke.py::test_async"
+  ]
+  ++ lib.optional stdenv.isDarwin [
+    "tests/unittest/conftest.py"
+    "tests/unittest/test_async.py"
+    "tests/unittest/test_async_session.py"
+    "tests/unittest/test_cookies.py"
+    "tests/unittest/test_curl.py"
+    "tests/unittest/test_headers.py"
+    "tests/unittest/test_impersonate.py"
+    "tests/unittest/test_requests.py"
+    "tests/unittest/test_upload.py"
+
+    # Hangs while testing
+    "tests/unittest/test_websockets.py"
   ];
```
This is what I'm seeing.